deepchem/models/keras_model.py

```diff
@@ -922,12 +934,20 @@ def _exec_with_session(self, f):
       with self.session.as_default():
         f()
 
-  def get_checkpoints(self):
-    """Get a list of all available checkpoint files."""
-    return tf.train.get_checkpoint_state(
-        self.model_dir).all_model_checkpoint_paths
+  def get_checkpoints(self, model_dir=None):
+    """Get a list of all available checkpoint files.
+
+    Parameters
+    ----------
+    model_dir: str, default None
+      Directory to get list of checkpoints from. Reverts to self.model_dir if None
+
+    """
+    if model_dir is None:
+      model_dir = self.model_dir
+    return tf.train.get_checkpoint_state(model_dir).all_model_checkpoint_paths
 
-  def restore(self, checkpoint=None):
+  def restore(self, checkpoint=None, restore_from=None):
```

Comment:
The argument should be called `model_dir` for consistency with `save_checkpoint()` and `get_checkpoints()`.

Reply:
Done